src/server/modules/sendResponse.ts

```diff
@@ -11,6 +11,7 @@ import {
   HL7_2_5_1, HL7_2_6, HL7_2_7, HL7_2_7_1, HL7_2_8
 } from 'node-hl7-client/hl7'
 import { PROTOCOL_MLLP_FOOTER, PROTOCOL_MLLP_HEADER } from '../../utils/constants.js'
+import type { ListenerOptions } from '../../utils/normalize'
```

Comment:
@jtarvainen  just needs to change to .js, but I will do that in the backend during merge